modules/microshift-restoring-data-backups.adoc

```diff
@@ -0,0 +1,45 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restoring-data-backups-manually_{context}"]
+= Restoring application data backups manually
+
+You can restore application data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When restoring a backup, you must use the entire file path.
```

Comment:
```suggestion
You can restore application data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When you restore a backup, you must use the entire file path.
```
From IBM Style: "When you use a verb phrase that begins with a present participle, such as _creating_, or past participle, such as _created_, at the beginning of a sentence, make sure that the verb phrase modifies the correct word. Failure to do so can result in a dangling modifier."